src/pages/ProjectWorkPage.js

```diff
@@ -65,47 +42,61 @@ const ProjectWorkPage = ({ match }) => {
   const project = example;
 
   return (
-    <div className="project-work-page">
-        <div className="row">
-          <div className="project-work-content">
-            <div className="project-work-flow">
-              <h1>Ready to work on this project?</h1>
-              <div className="justify-content-center d-flex flex-wrap align-items-center">
-                <ProjectFlow className="flow-images project-work-flow" />
+    <Container id="project-work-page" className="justify-content-md-center">
```

Comment:
I think we should just get rid of all this bootstrap stuff (mostly just container) bc it adds a bunch of styling we don't want, like the margins are huge on smaller screens (<992). also bootstrap is my enemy lmao
![image](https://user-images.githubusercontent.com/24930524/87859536-d7278e00-c903-11ea-9695-ca37bb1fbad2.png)

Reply:
i ended up just adjusting the specific breakpoints because I styled everything around the container :'(